### DISCUSSION TITLE
Get, Post, Put, Delete, Patch options

### DISCUSSION THREAD

**Discussion by II11II** [serverpod/serverpod#360] [Q&A] [answered]
How to create endpoints for each options?

**Comment by vlidholt** [comment, marked as answer]
This video may be a good starting point for understanding how you read/write to the database:
https://www.youtube.com/watch?v=93NgZLDB5pM

In Serverpod all calls are made with http post, unless you are using the Relic webserver. You would typically only use Relic if you want to serve an old fashioned web site or interact with other services through web-hooks.

**Reply by II11II** [reply]
Is there any sample code or docs ?
